Detectors/TRD/workflow/src/TRDGlobalTrackingSpec.cxx

```diff
@@ -370,6 +371,35 @@ void TRDGlobalTracking::run(ProcessingContext& pc)
   std::iota(trackIdxArray.begin(), trackIdxArray.end(), 0);
   std::sort(trackIdxArray.begin(), trackIdxArray.end(), [tracksOutRaw](int lhs, int rhs) { return tracksOutRaw[lhs].getCollisionId() < tracksOutRaw[rhs].getCollisionId(); });
 
+  std::vector<uint16_t> pileUpDist;
+  bool ft0Seen = false;
+  if (mTrkMask[GTrackID::FT0]) { // pile-up tagging was requested
+    auto ft0recPoints = inputTracks.getFT0RecPoints();
+    auto trdTriggers = inputTracks.getTRDTriggerRecords();
+    long maxDiff = mTracker->Param().rec.trd.pileupNBC;
+    ft0Seen = ft0recPoints.size() > 0;
+    pileUpDist.resize(trdTriggers.size(), 0);
```

Comment:
Ok, so you set it to 0 by default. That way, in case FT0 is available but no pileup collision was seen we use the nominal very small error for the TRD time.
In case FT0 is not available (and only in that case) you set the error to `mTracker->Param().rec.trd.pileupNBC`.
This looks good to me

Reply:
yes, and I am increasing (in the update to come) the nominal TRD error to 5 ns (since it is used as a gaussian error in TOF matching).